api/internal/localizer/localizer.go

```diff
@@ -0,0 +1,17 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package localizer
+
+import (
+	"log"
+
+	"sigs.k8s.io/kustomize/kyaml/errors"
+	"sigs.k8s.io/kustomize/kyaml/filesys"
+)
+
+func cleanDst(newDir string, fSys filesys.FileSystem) {
```

Comment:
What benefit does this function serve over just having calling `fSys.RemoveAll(newDir)`? This seems to be adding an extra layer without a clear benefit.

Reply:
I moved it into a function to log it, and I log it because of my comment below: https://github.com/kubernetes-sigs/kustomize/pull/4652#discussion_r937089621